site/pages/join-us/badger-testimonials-slice/index.js

```diff
@@ -1,65 +1,56 @@
 // @flow
-import React from 'react';
+import React, { useState } from 'react';
 import SwipeableViews from 'react-swipeable-views';
 import styles from './style.css';
 import ClientOnly from '../../../components/clientOnly';
-import Navigator from './navigator';
+import Navigator from '../../../components/navigator';
 import Testimonial from './testimonial';
 
 import testimonials from './testimonials';
 
-type BadgerTestimonialState = {
-  currentIndex: number,
-};
-
-type BadgerTestimonialProps = {
+type Props = {
   navPositionBottom?: boolean,
 };
 
-class BadgerTestimonialsSlice extends React.Component<
-  BadgerTestimonialProps,
-  BadgerTestimonialState,
-> {
-  state = {
-    currentIndex: 0,
-  };
-
-  setPage = (idx: number) => {
-    this.setState({
-      currentIndex: idx,
-    });
-  };
-
-  render() {
-    const { currentIndex } = this.state;
-    const { navPositionBottom } = this.props;
-
-    return (
-      <div className={styles.testimonials}>
-        <div className={styles.container}>
-          <ClientOnly>
-            {!navPositionBottom && <Navigator currentIndex={currentIndex} onClick={this.setPage} />}
-            <div className={navPositionBottom ? styles.content__bottomNav : styles.content}>
-              <SwipeableViews index={currentIndex} onChangeIndex={this.setPage}>
-                {testimonials.map(t => (
-                  <Testimonial
-                    key={t.content}
-                    content={t.content}
-                    author={t.author}
-                    role={t.role}
-                    icon={t.icon}
-                  />
-                ))}
-              </SwipeableViews>
-              {navPositionBottom && (
-                <Navigator currentIndex={currentIndex} onClick={this.setPage} navPositionBottom />
-              )}
-            </div>
-          </ClientOnly>
-        </div>
+const BadgerTestimonialsSlice = ({ navPositionBottom }: Props) => {
```

Comment:
Potentially could add a `renderNavigator` function to remove some of the duplication here.

```jsx
const renderNavigator = () => (
  <Navigator
    currentIndex={currentIndex}
    onChange={setPage}
    navPositionBottom
    maxIndex={testimonials.length - 1}
  />	      
);
```
```jsx
{!navPositionBottom && renderNavigator()}
```